cloudformation/cloudcam.yml

```diff
@@ -506,55 +487,55 @@ Resources:
           JANUS_HOSTED_ZONE_DOMAIN: !Ref JanusHostedZoneDomain
           JANUS_INSTANCE_NAME_PREFIX: !Ref JanusInstanceNamePrefix
   JanusKmsKey:
-    Type: "AWS::KMS::Key"
+    Type: AWS::KMS::Key
     Properties:
       Enabled: true
       EnableKeyRotation: false
       KeyPolicy:
-        Version: '2012-10-17'
-        Id: 'janus-kms'
+        Version: 2012-10-17
+        Id: janus-kms
         Statement:
-          - Effect: 'Allow'
+          - Effect: Allow
             Principal:
               AWS: !Sub 'arn:aws:iam::${AWS::AccountId}:root'
             Action:
-              - "kms:*"
+              - 'kms:*'
             Resource: '*'
-          - Effect: 'Allow'
+          - Effect: Allow
             Principal:
               AWS: !Sub 'arn:aws:iam::${AWS::AccountId}:user/cloudcam-ops'
             Action:
-              - "kms:Create*"
-              - "kms:Describe*"
-              - "kms:Enable*"
-              - "kms:List*"
-              - "kms:Put*"
-              - "kms:Update*"
-              - "kms:Revoke*"
-              - "kms:Disable*"
-              - "kms:Get*"
-              - "kms:Delete*"
-              - "kms:ScheduleKeyDeletion"
-              - "kms:CancelKeyDeletion"
+              - 'kms:Create*'
```

Comment:
is it right that this user has full KMS privs on `*`?

Reply:
Seems so since KeyPolicy only applies to the key in question and there's no way to put a reference to the key not yet created into KeyPolicy which is a parameter for it. (Also all the examples from KMS docs do the same Resource: '*' thing and testing seems to confirm that permissions given by this KeyPolicy only apply to the janus key and not the others.)

Reply:
Removed all permission except kms:Encrypt since it's the only one necessary right now